elixir/apps/web/lib/web/live/policies/show.ex

```diff
@@ -116,13 +116,20 @@ defmodule Web.Policies.Show do
       <:col :let={flow} label="EXPIRES AT">
         <.relative_datetime datetime={flow.expires_at} />
       </:col>
-      <:col :let={flow} label="CLIENT (IP)">
+      <:col :let={flow} label="CLIENT, ACTOR (IP)">
```

Comment:
Good fix, I think this column may make sense to be first or second now

Reply:
@jamilbk we show them in multiple places, so it will depend on what view you see. But yeah, it's one of the first ones here (only date comes first).